native_client/swift/stt_ios_test/SpeechRecognitionImpl.swift

```diff
@@ -8,149 +8,72 @@
 //  Copyright © 2021 Coqui GmbH
 import Foundation
 import AVFoundation
-import AudioToolbox
 import Accelerate
 
 import stt_ios
 
-struct FillComplexInputParm {
-    var source: UnsafeMutablePointer<Int8>
-    var sourceSize: UInt32
-};
-
-class SpeechRecognitionImpl : NSObject, AVCaptureAudioDataOutputSampleBufferDelegate {
+class SpeechRecognitionImpl {
     private var model: STTModel
     private var stream: STTStream?
 
-    private var captureSession = AVCaptureSession()
+    private let modelFeedInterval = 0.1
+
     private var audioData = Data()
+    private var timer: Timer? = nil
+    private var audioInput: AudioInput? = nil
+    private var bufferQueue: [[Int16]] = [[Int16]]()
```

Comment:
using `UnsafeBufferPointer<Int16>` for queue elements resulted in really bad artifacts in the audio somehow, so I use just an Int16 array